docs/recipes/ImplementingUndoHistory.md

```diff
@@ -0,0 +1,181 @@
+# Implementing Undo History
+
+To add undo functionality (or any other functionality) to your existing
+reducers, you need to create a higher-order reducer, which is a function
```

Comment:
Yeah no problem. Let's leave it hanging here for now; if you realize you don't have time, it's OK too—just let me know here in the comments so I can finish it.